code/go/0chain.net/blobbercore/handler/storage_handler.go

```diff
@@ -376,16 +375,15 @@ func (fsh *StorageHandler) GetFileStats(ctx context.Context, r *http.Request) (i
 		return nil, common.NewError("invalid_parameters", "Path is not a file.")
 	}
 
-	result := make(map[string]interface{})
-	result = fileref.GetListingData(ctx)
+	result := fileref.GetListingData(ctx)
 	stats, _ := stats.GetFileStats(ctx, fileref.ID)
 	wm, _ := writemarker.GetWriteMarkerEntity(ctx, fileref.WriteMarker)
 	if wm != nil && stats != nil {
 		stats.WriteMarkerRedeemTxn = wm.CloseTxnID
 	}
 	var statsMap map[string]interface{}
-	statsBytes, err := json.Marshal(stats)
-	err = json.Unmarshal(statsBytes, &statsMap)
+	statsBytes, _ := json.Marshal(stats)
```

Comment:
a simple marshal and unmarshal from struct, probably won't err, but I'm open to suggestion if we should still check for error.